README.md

```diff
@@ -61,9 +60,9 @@ To build with CUDA:
 
     cmake -DUSE_CUDA=ON ..
 
-To build with HSA, old DPDK and debug symbols (CHIME):
+To build with old DPDK and debug symbols (CHIME):
 
-    cmake -DRTE_SDK=/opt/dpdk-stable-16.11.4/ -DRTE_TARGET=x86_64-native-linuxapp-gcc -DUSE_OLD_DPDK=ON -DUSE_HSA=ON -DCMAKE_BUILD_TYPE=Debug ..
+    cmake -DRTE_SDK=/opt/dpdk-stable-16.11.4/ -DRTE_TARGET=x86_64-native-linuxapp-gcc -DUSE_OLD_DPDK=ON -DCMAKE_BUILD_TYPE=Debug ..
```

Comment:
This build no longer makes any sense without HSA/18.04 support.   Maybe add the complete CHORD build here instead.   Note at some point I want to make all of these auto detect so it's less annoying to work out what `cmake` command to use.